zerver/models/streams.py

```diff
@@ -180,6 +180,7 @@ def is_history_public_to_subscribers(self) -> bool:
     # * "realm" and "recipient" are not exposed to clients via the API.
     API_FIELDS = [
         "date_created",
+        "deactivated",
```

Comment:
This is an API change that needs to be discussed and approved in the "API design" stream to coordinate with non-web clients. You can see the recent thread I started for a good example.

Reply:
Started https://chat.zulip.org/#narrow/stream/378-api-design/topic/archived.20streams/near/1720667 for this.